torchvision/models/detection/generalized_rcnn.py

```diff
@@ -97,7 +99,7 @@ def forward(self, images, targets=None):
             features = OrderedDict([('0', features)])
         proposals, proposal_losses = self.rpn(images, features, targets)
         detections, detector_losses = self.roi_heads(features, proposals, images.image_sizes, targets)
-        detections = self.transform.postprocess(detections, images.image_sizes, original_image_sizes)
+        detections = postprocess(detections, images.image_sizes, original_image_sizes, self.training)
```

Comment:
I see this as a limitation, as now users can't customize the postprocessing themselves, while before it was possible to replace the transform with `model.transform = MyCustomTransform()` and it would work

Reply:
Yes! You're right. I didn't think out of the bounding box's perspective and indeed it's different for mask and keypoints, for which user choice seem to be a nice option